akka-stream/src/main/scala/akka/stream/scaladsl/Graph.scala

```diff
@@ -627,6 +627,75 @@ final class Broadcast[T](val outputPorts: Int, val eagerCancel: Boolean) extends
 
 }
 
+object WireTap {
+  private val singleton = new WireTap[Nothing]
+  def apply[T]() = singleton.asInstanceOf[WireTap[T]]
```

Comment:
Explicit return type and add scaladoc, maybe just a `@see [[WireTap]]` since there are details there.